### PR TITLE
(PC-11065) search: Do not index stopwords from `Offer.description`

### PR DESCRIPTION
**Pull request by dbaty**
The "description" field is the largest indexed field. Removing
stopwords would divide by 10 the space it takes on App Search. We lose
the possibility of finding in the description phrases that contain
stopwords, but that feature has been deemed unimportant and the
compromise looks reasonable.

---

Lien vers le ticket Jira : https://passculture.atlassian.net/browse/PC-11065

### CODE REVIEW

File: tests/core/search/test_serialize_appsearch.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def test_serialize_offer():
         "artist": "Author Performer Speaker Stage Director",
         "category": "LIVRE",
         "date_created": offer.dateCreated,
-        "description": "Un livre qu'il est bien pour le lire",
+        "description": "livre bien lire",
```
Comment (antoinewg) [inline tests/core/search/test_serialize_appsearch.py:44]: 👍

Comment (antoinewg) [reply tests/core/search/test_serialize_appsearch.py:44]: On peut rajouter une majuscule dans un mot, pour tester que c'est bien mis en minuscule au passage ?

**Review by antoinewg — APPROVED** [review]
top

File: tests/core/search/test_serialize_appsearch.py (hunk 1 above)
Comment (dbaty) [reply tests/core/search/test_serialize_appsearch.py:44]: yep, c'est fait

**Pull request merged**